namerd/main/src/main/scala/io/buoyant/namerd/NamerdAdmin.scala

```diff
@@ -20,6 +27,7 @@ class NamerdAdmin(app: App, config: NamerdConfig, namerd: Namerd) extends Admin(
     )),
     "/" -> new DtabListHandler(namerd.dtabStore),
     "/delegator.json" -> new DelegateApiHandler(namerd.namers.toSeq),
-    "/dtab/" -> new DtabHandler(namerd.dtabStore)
+    "/dtab/" -> new DtabHandler(namerd.dtabStore),
+    "/bound-names.json" -> new BoundNamesHandler(enumeratingNamers.toSeq)
```

Comment:
should this be on the admin or as a part of the HTTP controller API?  I'd expect it to be on the controller api.